tpp-app/src/services/utils/helpers.js

```diff
@@ -197,4 +185,20 @@ export const GETStoredYears = async () => {
     }
 
     return storedYears;
+}
+
+/** 
+* Gets the full current date as a string in the format of "2022-1-1"
+* @returns a string representing the current date
+*/
+
+export function getFullCurrentDateString() {
```

Comment:
i believe `getDateString(new Date())` will do what this helper does? also not sure if this is intended, but i don't think this helper takes into account that getMonth is 0-indexed in javascript (i.e. January = 0, February = 1, etc.)

Reply:
rip lmao

Reply:
WOuld it be fine if I just wrote a comment that the dates are 0-indexed?